README.md

```diff
@@ -34,11 +34,13 @@ classDef root fill:#8338ec,stroke:#8338ec,color:#fff;
 classDef default fill:#3a86ff,stroke:#3a86ff,color:#fff;
 classDef legacy fill:#3f506c,stroke:#3f506c,color:#fff;
 ```
-How scuttlebutt works with meta-feeds - each device now has a `root` feed,
-whose sole responsibility is to announce (point to) sub-feeds that you publish content to.
+How scuttlebutt works with metafeeds - each device now has a `root` metafeed,
+whose sole responsibility is to announce (point to) subfeeds that you publish
+content to. A subfeed can also be a metafeed, which then allows the existence of
+"sub-subfeeds".
 
-This means that when you first meet a peer you can replicate their `root` feed
-and (having discovered their subfeeds), replicate just their `aboutMe` and `contacts` feeds
+This means that when you first meet a peer you can replicate their `root` metafeed
+and, having discovered their subfeeds, replicate just their `aboutMe` and `contacts` feeds
```

Comment:
Maybe we should emphasize that these subfeeds are just examples of what you might want.

 replicate just the subfeeds you care about e.g. Aboutme...